blueprints/cloud-operations/network-dashboard/dashboards/quotas-utilization.json

```diff
@@ -375,31 +411,34 @@
           }
         },
         "width": 6,
-        "yPos": 20
+        "xPos": 6,
+        "yPos": 16
       },
       {
         "height": 4,
         "widget": {
-          "title": "ip_addresses_per_subnet_utilization",
+          "title": "firewalls_per_project_usage",
           "xyChart": {
             "chartOptions": {
               "mode": "COLOR"
             },
             "dataSets": [
               {
                 "minAlignmentPeriod": "60s",
-                "plotType": "LINE",
+                "plotType": "STACKED_BAR",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
+                  "apiSource": "DEFAULT_CLOUD",
                   "timeSeriesFilter": {
                     "aggregation": {
                       "alignmentPeriod": "60s",
+                      "crossSeriesReducer": "REDUCE_SUM",
+                      "groupByFields": [
+                        "metric.label.\"project\""
+                      ],
                       "perSeriesAligner": "ALIGN_MEAN"
                     },
-                    "filter": "metric.type=\"custom.googleapis.com/ip_addresses_per_subnet_utilization\" resource.type=\"global\"",
-                    "secondaryAggregation": {
-                      "alignmentPeriod": "60s"
-                    }
+                    "filter": "metric.type=\"custom.googleapis.com/firewalls_per_project_usage\" resource.type=\"global\""
```

Comment:
I think the metric should be "firewalls_per_project_vpc_usage":
                    "filter": "metric.type=\"custom.googleapis.com/firewalls_per_project_vpc_usage\" resource.type=\"global\""

Reply:
y